HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: default
+title: PROJECT
+parent: Hyperledger Improvement Proposals
+---
+
+# Sponsor(s)
+- Alex Popowycz (Hedera) [<a@hedera.com>](mailto:a@hedera.com)
+- Samson Goddy (Hedera) [<samson@hedera.com>](mailto:samson@hedera.com)
+- Richard Bair (Hashgraph) [<richard@hashgraph.com>](mailto:richard@hashgraph.com)
+- Leemon Baird (Independent) [<leemon@swirldslabs.com>](mailto:leemon@swirldslabs.com)
+- Hendrik Ebbers (Hashgraph) [<hendrik.ebbers@hashgraph.com>](mailto:hendrik.ebbers@hashgraph.com)
+- May Chan (HashPack) [<may@hashpack.app>](mailto:may@hashpack.app)
+- Shyam Nagarajan (IBM) [<shyam@us.ibm.com>](mailto:shyam@us.ibm.com)
+- Suma Nair (IBM) [<sumapnair@us.ibm.com>](mailto:sumapnair@us.ibm.com)
+- Katerina Sanchez-Schilling (DLT Science Foundation) [<Katerina.Sanchez-Schilling@dltscience.org>](mailto:Katerina.Sanchez-Schilling@dltscience.org)
+- Paolo Tasca (DLT Science Foundation) [<pt@dsf.xyz>](mailto:pt@dsf.xyz)
+- Nikhil Vadgama (DLT Science Foundation) [<nv@dsf.xyz>](mailto:nv@dsf.xyz)
+
+# Abstract
+This proposal outlines the plan to move Hedera’s core network software to the Linux Foundation’s Decentralized Trust Foundation.
```

Comment:
would be good to clarify what `core network software` means, in particular whether this will provide a complete set of components to stand up a Hedera network?